airbyte-integrations/base-python/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM airbyte/base:dev
+COPY --from=python:3.7-slim / /
```

Comment:
is this the recommended way?

Reply:
I'll look into this more. Charles was doing this for his PR and I thought it was clever due to how simple the command is. Not sure if it's the best/recommended.